jina/parsers/orchestrate/pod.py

```diff
@@ -103,3 +103,11 @@ def mixin_pod_parser(parser):
         dest='port_monitoring',
         help=f'The port on which the prometheus server is exposed, default port is {__default_port_monitoring__} ',
     )
+
+    gp.add_argument(
+        '--retries',
+        type=int,
+        default=None,  # default prometheus server port
```

Comment:
it should be 3 right? plus wrong comment

Reply:
Wrong comment, true, thanks for the catch.

But I would leave the default to `None`, which allows to keep the previous policy in place: Try at least three times, but if there are more than 3 replicas, try every replica at least once. That seems reasonable to me, because otherwise there might be a healthy replica but the call still fails. If the default is 3, we don't know if it was user set, and can't implement the default policy.

An alternative would be to always try every replica at least once, regardless of what the user puts in (`actual_num_retries = max(retries, num_replicas)`). But it feels a bit off to me to take that control away from the user.